### PR TITLE
App-293: Fix apach-rat-check warnings on latest develop

### PR DESCRIPTION
**Pull request by maxml**

### CODE REVIEW

File: verifiersdemo/pom.xml [hunk 1]
```diff
@@ -102,6 +102,281 @@
...
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKAppEvents.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKGraphErrorRecoveryProcessor.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKConstants.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/EndpointOwnershipDemo.xcworkspace/contents.xcworkspacedata
```
Comment (Slash32) [inline verifiersdemo/pom.xml:260]: This does not seem to be a thirdparty component, thus should not be excluded.

File: verifiersdemo/pom.xml [hunk 2]
```diff
@@ -102,6 +102,281 @@
...
+                        <exclude>
+                            /source/objc/FBSDKShareKit.framework/Headers/FBSDKSharePhoto.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKShareKit.framework/Headers/FBSDKShareOpenGraphContent.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKShareKit.framework/Headers/FBSDKShareConstants.h
+                        </exclude>
+                        <exclude>/source/android/kits.properties</exclude>
```
Comment (Slash32) [inline verifiersdemo/pom.xml:364]: Here and below the same as above.

**Review by Slash32 — CHANGES_REQUESTED** [review]

File: verifiersdemo/pom.xml (hunk 1 above)
Comment (am1kk) [reply verifiersdemo/pom.xml:260]: As a rule, files with the same extension doesn't have license header.

**Review by am1kk — APPROVED** [review]

File: verifiersdemo/pom.xml (hunk 2 above)
Comment (maxml) [reply verifiersdemo/pom.xml:364]: These files must be excluded. These are generated files, as I remember.

**Review by vadimol — APPROVED** [review]
LGTM

File: verifiersdemo/pom.xml (hunk 1 above)
Comment (vadimol) [reply verifiersdemo/pom.xml:260]: We can discuss it locally and provide a fix in the separate PR. Let's accept this now as a workaround. It is some sort of blocker.

**Pull request merged**